### ISSUE TITLE
[Feat]: migrate python.d/postgres to go.d/postgres

### ISSUE THREAD

**Issue by ilyam8** [netdata/netdata#12883]
### Problem

- All python.d collectors don't support service discovery in K8s. 
- We need a Postgres collector to monitor our Cloud infra (and we need to migrate it anyway).

### Description

The plan is to have Postgres collector written in go.

### Importance

must have

### Value proposition

1. ability to collect metrics from Posgres instances (both inside and outside to a K8s cluster) will be unblocked.
2. minus one more dependency (psycopg2).

**Comment by ilyam8** [comment]
Closing in favor of #13387
